### PR TITLE
proto/executor: add Kill executor

### PR DESCRIPTION
**Pull request by pingyu**
Add "Kill" executor to kill connection of remote TiDB instance.

This is for pingcap/tidb#17649.

### CODE REVIEW

**Review by SunRunAway — COMMENTED** [review]
LGTM

File: proto/executor.proto [hunk 1]
```diff
@@ -123,6 +125,11 @@ message Limit {
 	optional Executor child = 2;
 }
 
+message Kill {
+	optional uint64 connID = 1 [(gogoproto.nullable) = false];
+	optional bool query = 2 [(gogoproto.nullable) = false];
```
Comment (breeswish) [inline proto/executor.proto:130]: Could you document the meaning of `query`?

**Review by breeswish — DISMISSED** [review]
The rest LGTM

**Review by breeswish — APPROVED** [review]

**Comment by pingyu** [comment]
I'm sorry for making a mistake caused by wrong protoc version. It's fixed now, PTAL~
@breeswish

**Review by breeswish — DISMISSED** [review]

**Comment by breeswish** [comment]
Thanks a lot!

**Review by breeswish — APPROVED** [review]

**Pull request merged**